### DISCUSSION TITLE
Roadblocks?

### DISCUSSION THREAD

**Discussion by bdrelling** [isFakeAccount/psnawp#7] [General]
Hey there! I'm looking into doing something similar, albeit with Swift. Have you hit any major roadblocks?

Great work so far!

**Comment by isFakeAccount** [comment]
The roadblock I have is that Sony disabled the endpoint for messages. Tustin from https://github.com/Tustin/psn-php was able to figure out how to get endpoints from PSN app using jailbroken iPhone but he has been MIA for a while.

I tried to do something similar with rooted android however the PSN app would detect that something is tampered with and nothing worked, unfortunately.

**Reply by bdrelling** [reply]
Bummer, and thanks for sharing your learning! I've since stopped trying to access this data, but I'm sure the work will continue to be valuable to others going forward.
